net/mwan3/Makefile

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright (C) 2006-2014 OpenWrt.org
+# Copyright (C) 2006-2017 OpenWrt.org
```

Comment:
@champtar ok i will drop this.
But some did some did not change this!
When should i update this?
My information is if i change the Makefile i have to update the year.